pkgs/native_toolchain_c/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 0.5.5-wip
 
 - Address analyzer info diagnostic about multi-line if requiring a block body.
+- Use unified classes instead of two `{OS,...}` and `{OS,,...}Impl`
```

Comment:
Maybe just mention bumping native_assets_cli and taking in the breaking changes from there.

(The current message has a type with double comma.)

Reply:
Done.